client/client.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/gobuffalo/pop/v6"
 	"github.com/gofrs/uuid"
 
-	jose "gopkg.in/square/go-jose.v2" // Naming the dependency jose is important for go-swagger to work, see https://github.com/go-swagger/go-swagger/issues/1587
+	"gopkg.in/square/go-jose.v2" // Naming the dependency jose is important for go-swagger to work, see https://github.com/go-swagger/go-swagger/issues/1587
```

Comment:
This change seems to conflict with the comment

Reply:
Yea IDK, my IDE (Goland) does this automatically. I think this is not needed anyways (because the package is named `jose` already), but we can remove the comment. OK?